pkg/config/path.go

```diff
@@ -43,21 +43,22 @@ func WithConfigFileFlag(flag string) Option {
 		var envPaths []string
 		var paths []string
 
+		// check PWD
 		if pwd := os.Getenv("PWD"); pwd != "" {
-			envPaths = []string{path.Join("$PWD", dotfile)}
-			paths = []string{path.Join(pwd, dotfile)}
+			envPaths = append(envPaths, path.Join("$PWD", dotfile))
+			paths = append(paths, path.Join(pwd, dotfile))
 		}
 
 		// check HOME
 		if home := os.Getenv("HOME"); home != "" {
```

Comment:
While we're at it, shouldn't we also rewrite this to use [`os.UserHomeDir`](https://golang.org/pkg/os/#UserHomeDir) and [`os.UserConfigDir`](https://golang.org/pkg/os/#UserConfigDir) so that it also works on Windows?

Reply:
Not sure what to do with `$PWD` above on Windows, but I'm pretty sure that doesn't work either.

Reply:
> While we're at it, shouldn't we also [insert improvement here]

... is a general feeling we can apply to `pkg/config`

In any case, let's see what we can do here